### PR TITLE
remove tab from comment

### PR DESCRIPTION
**Pull request by niradler**
# In This PR

- checking bug related to skipping comment

## Pictures/videos

- [x] I've reviewed my own code

### CODE REVIEW

File: src/suggestFix.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ const getCommentStringByFileName = (file: string): string => {
     return specialFilesDict[fileType] || '#';
 };
 
-const generateSkipComment = (checkId: string, file: string) => `\t${getCommentStringByFileName(file)} checkov:skip=${checkId}: ADD REASON\n`;
+const generateSkipComment = (checkId: string, file: string) => `${getCommentStringByFileName(file)} checkov:skip=${checkId}: ADD REASON\n`;
```
Comment (tsmithv11) [inline src/suggestFix.ts:35]: Can we make it 2 spaces instead?

**Review by tsmithv11 — APPROVED** [review]

File: src/suggestFix.ts (hunk 1 above)
Comment (niradler) [reply src/suggestFix.ts]: @tsmithv11 sure replaced do you want to proceed?

Comment (tsmithv11) [reply src/suggestFix.ts]: @niradler looks good. We need to do the same for fixes, but this is good for me.

**Review by yuvalyacoby — APPROVED** [review]

**Comment by niradler** [comment]
@tsmithv11 do you have push permission?

**Comment by tsmithv11** [comment]
> @tsmithv11 do you have merge permission?

@niradler no. I'll add others

**Review by achiar99 — APPROVED** [review]

**Pull request merged**